server/src/main/java/org/opensearch/indices/IndicesService.java

```diff
@@ -1993,6 +1993,9 @@ public Iterator<Setting<?>> settings() {
      * @param defaultRefreshInterval value of cluster default index refresh interval setting
      */
     private static void validateRefreshIntervalSettings(TimeValue minimumRefreshInterval, TimeValue defaultRefreshInterval) {
+        if (defaultRefreshInterval.millis() < 0) {
```

Comment:
Is this to allow setting cluster level default refresh interval to -1?

Reply:
Yes @sachinpkale